cms/urls.py

```diff
@@ -75,13 +75,13 @@
     url(r'^signin$', 'login_page', name='login'),
     url(r'^request_course_creator$', 'request_course_creator'),
 
-    url(r'^course_team/{}/(?P<email>.+)?$'.format(COURSELIKE_KEY_PATTERN), 'course_team_handler'),
+    url(r'^course_team/{}(?:/(?P<email>.+))?$'.format(COURSELIKE_KEY_PATTERN), 'course_team_handler'),
```

Comment:
@cahrens there are two groups: outermost is non-cpaturing, innermost is capturing and named. E.g. in `course_team/course_id/staff@example.com`, first (non-capturing) group would contain `/staff@example.com`, while second (capturing, named) - `staff@example.com`.

The reason behind this is to make entire part after course_id optional, but still capturing email and sending it to view.